src/analysis/retail/deathknight/blood/CombatLogParser.ts

```diff
@@ -104,13 +104,13 @@ class CombatLogParser extends CoreCombatLogParser {
     // guide stuff
     deathStrike: DeathStrike,
     bloodShield: BloodShield,
-    aplCheck: AplCheck,
 
     // normalizers
     deathStrikeNormalizer: DeathStrikeLinkNormalizer,
     bloodShieldNormalizer: BloodShieldNormalizer,
     resourceOrderNormalizer: ResourceOrderNormalizer,
     boneShieldOrderNormalizer: BoneShieldOrderNormalizer,
+    ExterminateCostNormalizer,
```

Comment:
This seems to be differnt from the standard we're using.
```suggestion
    exterminateCostNormalizer: ExterminateCostNormalizer,
```

Reply:
the `dependencies` array needs to be unique by name except that you also have to match the name of superclasses you're overriding. ive started using the shorthand a bit. if people don't like it i can switch back to using `lowerCamelCase: CamelCase` instead